pkgs/by-name/st/steam-play-none/package.nix

```diff
@@ -0,0 +1,43 @@
+{ fetchFromGitHub
+, stdenvNoCC
+, lib
+, bash
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  name = "Steam-Play-None";
+  version = "0-unstable-2022-12-15";
+  src = fetchFromGitHub {
+    repo = finalAttrs.name;
```

Comment:
```suggestion
    repo = "Steam-Play-None";
```

Reply:
It seems it's not case sensitive anyway, so we'll keep finalAttrs.

Reply:
https://github.com/NixOS/nixpkgs/issues/277994